packages/react-dom/src/__tests__/ReactDOMServerSelectiveHydration-test.internal.js

```diff
@@ -266,9 +270,9 @@ describe('ReactDOMServerSelectiveHydration', () => {
       resolve();
       await promise;
     });
-    // After the click, we should prioritize D and the Click first,
+    // After the click, we should prioritize hydrating D
```

Comment:
Why?

Reply:
I removed "and the click first" since we dont replay clicks now.
D was clicked though which increased its priority.